Description: do not use libc++abi during build
 the main reason being it's not even there
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2015-09-05
--- a/Makefile
+++ b/Makefile
@@ -251,8 +251,7 @@
 	   test/cctest/cctest.gyp \
 	   test/unittests/unittests.gyp tools/gyp/v8.gyp \
 	   tools/parser-shell.gyp testing/gmock.gyp testing/gtest.gyp \
-	   buildtools/third_party/libc++abi/libc++abi.gyp \
-	   buildtools/third_party/libc++/libc++.gyp samples/samples.gyp \
+	   samples/samples.gyp \
 	   src/third_party/vtune/v8vtune.gyp src/d8.gyp
 
 # If vtunejit=on, the v8vtune.gyp will be appended.
